spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -583,20 +581,24 @@ The Jakarta Data implementation automatically recognizes the query annotations i
 
 === Parameter-based automatic query methods
```

Comment:
This section does not include Query by Method Name.  It says, "the Query by Method Name pattern is discussed separately", and  there is a separate section, at the same level, for that:

```
=== Query by Method Name
```

It would be confusing to call this section Automatic query methods when it is not the place to find information on both types of Automatic query method, and is only intended for Parameter-based conditions on automatic query methods.

Reply:
Ah good point, I think I glossed over the part in parenthesis, but it's pretty clearly stated.